catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -548,38 +357,20 @@
         <argument ref="attributeRegistry"/>
     </bean>
 
-    <bean id="searchFormsApplication" class="org.codice.ddf.catalog.ui.forms.SearchFormsApplication">
+    <bean id="searchFormsApplication"
+          class="org.codice.ddf.catalog.ui.forms.SearchFormsApplication">
         <argument ref="catalogFramework"/>
         <argument ref="filterBuilder"/>
         <argument ref="templateTransformer"/>
         <argument ref="endpointUtil"/>
     </bean>
 
-    <reference id="logoutService" interface="org.codice.ddf.security.logout.service.LogoutService"/>
-
-    <bean id="logoutApplication" class="org.codice.ddf.catalog.ui.security.LogoutApplication">
-        <argument ref="logoutService"/>
-    </bean>
-
-    <reference id="platformConfigurationService"
-               interface="org.codice.ddf.configuration.service.PlatformUiConfigurationService"/>
-
-    <bean id="platformApplication"
-          class="org.codice.ddf.catalog.ui.configuration.PlatformUiConfigurationApplication">
-        <argument ref="platformConfigurationService"/>
-    </bean>
-
-    <reference id="sessionManagementService"
-               interface="org.codice.ddf.security.session.management.service.SessionManagementService"/>
-
-    <bean id="sessionManagementApplication"
-          class="org.codice.ddf.catalog.ui.session.SessionManagementApplication">
-        <argument ref="sessionManagementService"/>
-    </bean>
-
-    <bean id="transformerDescriptorApplication" class="org.codice.ddf.catalog.ui.transformer.TransformerDescriptorApplication">
-        <property name="descriptors" ref="transformerDescriptors"/>
-    </bean>
+    <!--
+    =============================================================
+    SPARK
+    =============================================================
+    -->
+    <!-- TODO Remember that the attachment parser was removed on purpose since nobody was using it -->
```

Comment:
❓ We had a reference to the attachment parser that nobody was using so I removed it. Want to ensure this is acceptable.

Reply:
Verified that Intrigue's uploader still works.